src/lib.rs

```diff
@@ -384,7 +384,7 @@ impl Config {
 
         let mut objects = Vec::new();
         let mut src_dst = Vec::new();
-        for file in self.files.iter() {
```

Comment:
I personally prefer `for x in foo.iter()` over `for x in &foo`, can this remain as it was?